maybe-rayon/src/lib.rs

```diff
@@ -226,6 +226,25 @@ impl<T: Send> MaybeParChunksMut<T> for [T] {
     }
 }
 
+#[cfg(feature = "parallel")]
+pub trait MaybeParWindows<T: Sync> {
```

Comment:
I think we need a variant of this trait under `#[cfg(not(feature = "parallel"))]` also